v2/css/main.css

```diff
@@ -868,3 +868,4 @@ hr.footer-sep {
   font-size: 65%;
   vertical-align: top;
 }
+
```

Comment:
Don't introduce such things. Be careful. Always read the diffs yourself.

Reply:
empty line removed.